recipe/meta.yaml

```diff
@@ -114,7 +118,7 @@ test:
     - if not exist %PREFIX%\\Library\\bin\\arrow.dll exit 1          # [win]
     - if not exist %PREFIX%\\Library\\bin\\arrow_dataset.dll exir 1  # [win]
     - if not exist %PREFIX%\\Library\\bin\\arrow_flight.dll exit 1   # [win]
-    - if not exist %PREFIX%\\Library\\bin\\arrow_python.dll exit 1   # [win]
+    # - if not exist %PREFIX%\\Library\\bin\\arrow_python.dll exit 1   # [win]
```

Comment:
See above comments about removing Python support.

Reply:
If we enable the test the build will simply fail during testing.
It seems that the feature for `ARROW_PYTHON` is deprecated.